challenge_where_is_this/README.md

```diff
@@ -0,0 +1,44 @@
+# Storing groceries
```

Comment:
This is the README of storing groceries. Please update the README with the correct challenge description and instructions